analyses/molecular-subtyping-HGG/02-HGG-molecular-subtyping-subset-files.R

```diff
@@ -259,3 +263,4 @@ snv_consensus_hotspot_maf <- snv_consensus_hotspot_maf %>%
 write_tsv(snv_consensus_hotspot_maf,
           file.path(subset_dir, "hgg_snv_maf.tsv.gz"))
 rm(snv_consensus_hotspot_maf)
+
```

Comment:
For line 113 here, let's add back methylation since you are adding methyl for the other samples.